index.html

```diff
@@ -1,5 +1,240 @@
+                </div>
+              </div>
+            </li>
+          </ul>
+        </div>
+      <div class="u-fixed-width">
+        <hr class="p-separator" />
+      </div>
+      <div class="row">
+        <div class="col-8">
+          <h2>Quick start</h2>
+          <p>
+            We recommend you use Ubuntu. On Mac or Windows, you can use <a href="https://multipass.run">Multipass</a> to give yourself Ubuntu VMs on demand. To install the <a href="https://snapcraft.io/microstack">MicroStack&nbsp;snap</a>:
+          </p>
+          <h3>Install MicroStack</h3>
+          <div class="p-notification--information">
+            <p class="p-notification__response">
+              <span class="p-notification__status">Information:</span>
+              MicroStack requires at least 16 GB of RAM and a multi-core processor.
+            </p>
+          </div>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" aria-label="sudo snap install microstack --classic --beta" value="sudo snap install microstack --classic --beta" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p>
+            Command <code>snap</code> not found? <a class="p-link--external" href="https://snapcraft.io/docs/core/install">Install snapd first.</a>
+          </p>
+          <h3>Configure networks and OpenStack databases</h3>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" aria-label="sudo microstack.init" value="sudo microstack.init" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p>
+            That’s it! Your OpenStack is ready.
+          </p>
+        </div>
+      </div>
+      <div class="u-fixed-width">
+        <hr class="p-separator" />
+      </div>
+      <div class="row">
+        <div class="col-8">
+          <h2>
+            Useful tips
+          </h2>
+          <h3>
+            Launch a VM
+          </h3>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" aria-label="microstack.launch test" value="microstack.launch test" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p>
+            Your VM is up and running now. You should see a message like:
+          </p>
+          <p><code>Access your server with 'ssh -i /home/ubuntu/.ssh/id_microstack cirros@10.20.20.3'</code></p>
+
+          <h3>
+            Try a different version, or a beta, or a daily build
+          </h3>
+          <p>
+            Snaps are published in <a class="p-link--external" href="https://docs.snapcraft.io/channels/551">channels</a> which are made up of a track (or major version), and an expected level of stability. Try snap info microstack to see what versions are currently published. You can run
```

Comment:
Should "snap info microstack" be in a code copyable block?

Reply:
I think a code block is ok, and I found another... thanks